### PR TITLE
deadbeef-statusnotifier-plugin: init at 1.6

### PR DESCRIPTION
**Pull request by kurnevsky**
###### Motivation for this change
StatusNotifier icon plugin for deadbeef. By default it supports only xembed icons.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -5749,6 +5749,12 @@
     githubId = 278013;
     name = "Tomasz Kontusz";
   };
+  kurnevsky = {
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:5752]: Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.

File: pkgs/applications/audio/deadbeef/plugins/statusnotifier.nix [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{ lib, stdenv, fetchurl, pkg-config, deadbeef, gtk3, perl, libdbusmenu-glib }:
+
+stdenv.mkDerivation rec {
+  pname = "deadbeef-statusnotifier-plugin";
+  version = "1.6";
+
+  src = fetchurl {
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/deadbeef/plugins/statusnotifier.nix:7]: Please use fetchFromGitHub.

File: pkgs/applications/audio/deadbeef/plugins/statusnotifier.nix [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ deadbeef gtk3 libdbusmenu-glib ];
+
+  buildFlags = [ "gtk3" ];
+
+  postPatch = [''
+    substituteInPlace tools/glib-mkenums \
+      --replace /usr/bin/perl "${perl}/bin/perl"
+  ''];
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/deadbeef/plugins/statusnotifier.nix:21]:
```suggestion
  postPatch = ''
    substituteInPlace tools/glib-mkenums \
      --replace /usr/bin/perl "${perl}/bin/perl"
  '';
```

File: pkgs/applications/audio/deadbeef/plugins/statusnotifier.nix [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    runHook preInstall
+    mkdir -p $out/lib/deadbeef
+    cp build/sni_gtk3.so $out/lib/deadbeef
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "DeaDBeeF StatusNotifier Plugin";
+    homepage = "https://github.com/vovochka404/deadbeef-statusnotifier-plugin";
+    license = licenses.gpl3;
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/deadbeef/plugins/statusnotifier.nix:33]: Do you know if the license is gpl3Only or gpl3Plus?

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (kurnevsky) [reply maintainers/maintainer-list.nix:4560]: Done.

File: pkgs/applications/audio/deadbeef/plugins/statusnotifier.nix (hunk 2 above)
Comment (kurnevsky) [reply pkgs/applications/audio/deadbeef/plugins/statusnotifier.nix]: Done.

File: pkgs/applications/audio/deadbeef/plugins/statusnotifier.nix (hunk 3 above)
Comment (kurnevsky) [reply pkgs/applications/audio/deadbeef/plugins/statusnotifier.nix]: Done.

File: pkgs/applications/audio/deadbeef/plugins/statusnotifier.nix (hunk 4 above)
Comment (kurnevsky) [reply pkgs/applications/audio/deadbeef/plugins/statusnotifier.nix]: Looks like it's `gpl3Plus`, at least headers mention this: https://github.com/vovochka404/deadbeef-statusnotifier-plugin/blob/c1a8a2d81d40d14a6f34f6b82a5bf154327e0e20/src/sni_timer.c#L12

So changed it to gpl3Plus.
